src/game/GameManager.h

```diff
@@ -53,6 +53,13 @@ class GameManager {
     void updateCollider(); // Updates CollisionHandler
     void updateMarines(const float delta); // Update marine actions
     void updateZombies(const float delta); // Update zombie actions
+    void updateTurrets(const float delta); // Update turret actions
+
+    // returns the list of zombies.
+    // Jamie, 2017-03-01.
+    const auto& getZombies() {
```

Comment:
Remove const here
I screwed up saying it should be const
I'm assuming you'd want to be able to change the contents of zombie manager through this getter